wcfsetup/install/files/lib/system/tagging/TagEngine.class.php

```diff
@@ -41,10 +41,12 @@ public function addObjectTags($objectType, $objectID, array $tags, $languageID,
 
         // remove tags prior to apply the new ones (prevents duplicate entries)
         if ($replace) {
-            $sql = "DELETE FROM wcf" . WCF_N . "_tag_to_object
-                    WHERE       objectTypeID = ?
-                            AND objectID = ?
-                            AND languageID = ?";
+            $sql = "DELETE FROM wcf" . WCF_N . "_tag_to_object tag_to_object
```

Comment:
Will have to recheck this query. I believe I forgot to adjust the code after testing in MySQL CLI.